docs/section22/Rule22-28.md

```diff
@@ -0,0 +1,48 @@
+
+# CHW&CW - Rule 22-28  
+
+**Schema Version:** 0.0.34    
+**Mandatory Rule:** True  
+**Rule ID:** 22-28  
+**Rule Description:** The baseline building design condenser-water pump shall be modeled as constant volume.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**90.1 Section Reference:** G3.1.3.11
+**Data Lookup:** None  
+**Evaluation Context:** Each HeatRejection 
+**Applicability Checks:**  
+
+1. B-RMI is modeled with at least one air-side system that is Type-7, 8, 12, 13, 7b, 8b, 12b.
+2. B-RMI is modeled with heat rejection loop
+
+**Function Call:**  
+
+1. get_heat_rejection_loops_connected_to_baseline_systems()
+
+**Applicability Checks:**  
+
+- The function get_heat_rejection_loops_connected_to_baseline_systems() returns a list of loops that are connected to Type 7,8,12,13,7b,8b,12b.  Get the list of applicable heat rejection loops: `heat_rejection_loop_ids = get_heat_rejection_loops_connected_to_baseline_systems(B_RMI)`
+
+- check applicability for each HeatRejection in the model: `for heat_rejection in B_RMI.heat_rejections:`
+
+  - check if heat_rejection fluid_loop is one of the applicable loops, if so, continue to the rule logic: `if heat_rejection.loop.id in heat_rejection_loop_ids: CONTINUE TO RULE LOGIC`
+  
+  - Else, rule is not applicable to the heat rejection: `else: RULE_NOT_APPLICABLE`
+
+  ## Rule Logic:  
+
+  - loop through the pumps in the RMI and find the pump that is connected to the loop: `for pump in B-RMI.pumps:`
+    - if the pump is the one connected to the heat_rejection_loop, continue to rule Assertion: `if pump.loop_or_piping == heat_rejection.loop:`
+
+      **Rule Assertion - HeatRejection:**
+
+      - Case 1: If the pump speed control equals FIXED_SPEED, PASS: `if ( pump.speed_control == FIXED_SPEED ): PASS`
+
```

Comment:
@KarenWGard  wondering if we should be checking loop flow control rather than pump speed control?
FuildLoopDesignandControl has a flow_cotrop option which includes 'fixed flow'
would that be more appropriate than fixed speed pump?. we might have discussed this before as well

Reply:
Should we check for both?

@mkarpman - thoughts?

Reply:
I agree with Supriya that we should be checking  FluidLoopDesignand Contrl.flow_control=FIXED_FLOW. I don't think we need to additionally check for pump.speed_control because it is not explicitly required in 90.1, plus I don't think it matters (at least in eQUEST) for the fixed flow loops.